### PR TITLE
Support for LoaderAccount

### PR DESCRIPTION
**Pull request by NorbertBodziony**
Got stuck on `generics_test.rs` 

Let me know if it even makes sense.

### CODE REVIEW

**Comment by armaniferrante** [comment]
Addresses https://github.com/project-serum/anchor/issues/643

File: lang/src/loader_account.rs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+/// for example, the [`ProgramAccount`](./struct.ProgramAccount.html). Namely,
+/// one must call `load`, `load_mut`, or `load_init`, before reading or writing
+/// to the account. For more details on zero-copy-deserialization, see the
+/// [`account`](./attr.account.html) attribute.
+///
+/// When using it's important to be mindful of any calls to `load` so as not to
+/// induce a `RefCell` panic, especially when sharing accounts across CPI
+/// boundaries. When in doubt, one should make sure all refs resulting from a
+/// call to `load` are dropped before CPI.
+pub struct LoaderAccount<'info, T: ZeroCopy + Owner> {
```
Comment (armaniferrante) [inline lang/src/loader_account.rs:27]: As suggested on Discord, let's rename `LoaderAccount` to `AccountLoader`.

Comment (NorbertBodziony) [reply lang/src/loader_account.rs]: fixed

**Review by armaniferrante — COMMENTED** [review]
LGTM. Last thing we need to do is add this to an integration test.

**Comment by NorbertBodziony** [comment]
Gonna work on this tmw.

**Comment by NorbertBodziony** [comment]
All should be done refactored `zero_copy` test + removed state and added CPI using `zero_copy`

**Pull request merged**